packages/addons/service/pcscd/package.mk

```diff
@@ -24,12 +24,7 @@ addon() {
   mkdir -p $ADDON_BUILD/$PKG_ADDON_ID/bin/
     cp -Pa $(get_install_dir pcsc-lite)/usr/sbin/pcscd $ADDON_BUILD/$PKG_ADDON_ID/bin/pcscd.bin
 
-  mkdir -p $ADDON_BUILD/$PKG_ADDON_ID/drivers/serial
-    cp -Pa $(get_build_dir ccid)/.$TARGET_NAME/src/.libs/libccidtwin.so $ADDON_BUILD/$PKG_ADDON_ID/drivers/serial
-
-  mkdir -p $ADDON_BUILD/$PKG_ADDON_ID/drivers/ifd-ccid.bundle/Contents/Linux/
-    cp -Pa $(get_build_dir ccid)/.$TARGET_NAME/src/.libs/libccid.so $ADDON_BUILD/$PKG_ADDON_ID/drivers/ifd-ccid.bundle/Contents/Linux/
-    cp -Pa $(get_build_dir ccid)/.$TARGET_NAME/src/Info.plist $ADDON_BUILD/$PKG_ADDON_ID/drivers/ifd-ccid.bundle/Contents
+  cp -a $(get_install_dir ccid)/storage/.kodi/addons/$PKG_ADDON_ID/drivers $ADDON_BUILD/$PKG_ADDON_ID
```

Comment:
Is this source path correct?

Reply:
I think so, yes:
```
install_pkg/ccid-1.4.28
install_pkg/ccid-1.4.28/.libreelec-package
install_pkg/ccid-1.4.28/etc
install_pkg/ccid-1.4.28/etc/reader.conf.d
install_pkg/ccid-1.4.28/etc/reader.conf.d/libccidtwin
install_pkg/ccid-1.4.28/storage
install_pkg/ccid-1.4.28/storage/.kodi
install_pkg/ccid-1.4.28/storage/.kodi/addons
install_pkg/ccid-1.4.28/storage/.kodi/addons/service.pcscd
install_pkg/ccid-1.4.28/storage/.kodi/addons/service.pcscd/drivers
install_pkg/ccid-1.4.28/storage/.kodi/addons/service.pcscd/drivers/serial
install_pkg/ccid-1.4.28/storage/.kodi/addons/service.pcscd/drivers/serial/libccidtwin.so
install_pkg/ccid-1.4.28/storage/.kodi/addons/service.pcscd/drivers/ifd-ccid.bundle
install_pkg/ccid-1.4.28/storage/.kodi/addons/service.pcscd/drivers/ifd-ccid.bundle/Contents
install_pkg/ccid-1.4.28/storage/.kodi/addons/service.pcscd/drivers/ifd-ccid.bundle/Contents/Info.plist
install_pkg/ccid-1.4.28/storage/.kodi/addons/service.pcscd/drivers/ifd-ccid.bundle/Contents/Linux
install_pkg/ccid-1.4.28/storage/.kodi/addons/service.pcscd/drivers/ifd-ccid.bundle/Contents/Linux/libccid.so
```

Reply:
Yes, it's correct. But it's a bit weird. It's unrelated to this PR, of course, but we shouldn't be hard coding `/storage/.kodi/addons/[service|virtual|whatever].[addonid]/` in multiple locations, which is what we seem to be doing. Perhaps something to clean up in a future PR...